<ui version="4.0" >
 <class>OmgGdalConverterBase</class>
 <widget class="QDialog" name="OmgGdalConverterBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Raster File Converter</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/om_logo.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="lblInputFile" >
     <property name="text" >
      <string>Input files</string>
     </property>
     <property name="buddy" >
      <cstring>pbnRemoveFile</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QGridLayout" >
     <item row="1" column="1" >
      <widget class="QPushButton" name="pbnRemoveFile" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/resources.qrc" >:/edittrash.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>30</width>
         <height>103</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="pbnSelectFile" >
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/resources.qrc" >:/fileopen.png</iconset>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="0" column="0" >
      <widget class="QListWidget" name="lstInputFiles" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="lblOutputDir" >
     <property name="text" >
      <string>Output folder</string>
     </property>
     <property name="buddy" >
      <cstring>pbnRemoveFile</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLineEdit" name="leOutputDir" />
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="pbnSelectDir" >
     <property name="minimumSize" >
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>26</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/fileopen.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="lblOutputType" >
     <property name="text" >
      <string>Output format</string>
     </property>
     <property name="buddy" >
      <cstring></cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QComboBox" name="cboOutputFileType" >
     <item>
      <property name="text" >
       <string>GeoTiff</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Arc/Info ASCII Grid</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OmgGdalConverterBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>374</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel" >
     <x>432</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OmgGdalConverterBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>306</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
